### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by doncem**
no major version jumps so up'ing the current one by `0.0.1`

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -36,6 +36,10 @@ allprojects {
     ruleSetFiles = files("$rootProject.projectDir/config/pmd/ruleset.xml")
   }
 
+  jacoco {
+    toolVersion = "0.8.1"
```
Comment (timja-kainos) [inline build.gradle]: this should be the default on new gradle

File: prepare-for-release.sh [hunk 2]
```diff
@@ -13,7 +13,7 @@ fi
 
 set -u
 
-sed -i '' -e "s|${CURRENT_VERSION}|${NEW_VERSION}|" $(find . -name "README.md" -o -name "gradle.properties" | xargs -n 1)
+sed -i '' "s|${CURRENT_VERSION}|${NEW_VERSION}|" $(find . -name "README.md" -o -name "gradle.properties" | xargs -n 1)
```
Comment (timja-kainos) [inline prepare-for-release.sh]: hmm I think this will break on mac with this change what OS are you on?

Comment (doncem) [reply prepare-for-release.sh]: mac

File: build.gradle (hunk 1 above)
Comment (doncem) [reply build.gradle]: that's what i've missed - gradle upgrade

File: prepare-for-release.sh (hunk 2 above)
Comment (timja-kainos) [reply prepare-for-release.sh]: tested your change works fine for me.
unsure why its needed but 👍

**Pull request merged**